src/borg/archive.py

```diff
@@ -931,7 +931,7 @@ def init_chunks(self):
         """
         # Explicitly set the initial hash table capacity to avoid performance issues
         # due to hash table "resonance"
-        capacity = int(len(self.repository) * 1.052 + 1)  # > len * 1.0 / HASH_MAX_LOAD (see _hashindex.c)
+        capacity = int(len(self.repository) * (1/0.99) + 1)  # > len * 1.0 / HASH_MAX_LOAD (see _hashindex.c)
```

Comment:
sure. i tried that long ago, but at least it did not work directly. but maybe i tried the wrong way...